tasks/main.yml

```diff
@@ -43,6 +50,27 @@
   notify: restart influxdb
   tags: [influxdb]
 
+- name: Disable HTTP query logging for influxdb
+  lineinfile:
+    path: /etc/influxdb/influxdb.conf
+    line: '  log-enabled = true'
+    insertafter: '\[http\]'
+    state: present
+  tags: [influxdb]
+
+- name: Set maxsize of rsyslog logrotate configuration
+  block:
+    - lineinfile:
+        path: /etc/logrotate.d/rsyslog
+        firstmatch: true
+        line: '        maxsize 65M'
+        insertafter: '\{'
+    - lineinfile:
+        path: /etc/logrotate.d/rsyslog
+        line: '        maxsize 65M'
+        insertafter: '\{'
+  tags: [influxdb1]
+
```

Comment:
@nemesisdesign are you sure we need this change here? 

> Modify /etc/logrotate.d/rsyslog to add maxsize 65M to both sections in that file

Reply:
yes because the logged queries end up in these files even though they're disabled